.travis.yml

```diff
@@ -47,7 +47,7 @@ before_script:
 # Use Docker if available (currently Linux only)
 # macOS/Linux compatible temp dir deleted on reboot (https://stackoverflow.com/a/31397073)
 - |
-  if which docker > /dev/null ; then
+  if command -v docker > /dev/null ; then
```

Comment:
interesting, i did not know about `command`

Reply:
same here. Thanks shellcheck!